src/RecipeRepository.js

```diff
@@ -20,9 +20,7 @@ class RecipeRepository {
   }
 
   filterByName(searchTerm) {
-    return this.recipes.filter(recipe => {
-      return recipe.name.toLowerCase().includes(searchTerm);
-    });
+    return this.recipes.filter(recipe => recipe.name.toLowerCase() === searchTerm.toLowerCase());
```

Comment:
This is cleaner for sure. I'm wondering though if it will break the functionality we had before where a search for "cake" returns recipes matching "cake" as well as recipes matching "cupcake" and "pancake"? Also we had the formatting of the search term happening in `scripts.js`: that's where the search string is being broken into its component words and being lowercased. We probably don't need to duplicate that lowercasing functionality, and it probably makes more sense to have it here in the class method.

Reply:
talked about this in zoooooom